server/src/main/java/org/elasticsearch/cluster/coordination/CoordinationDiagnosticsService.java

```diff
@@ -361,7 +511,8 @@ private CoordinationDiagnosticsResult getResultOnNoMasterEligibleNodes(MasterHis
         CoordinationDiagnosticsDetails details = getDetails(
             explain,
             localMasterHistory,
-            coordinator.getClusterFormationState().getDescription()
+            null,
+            Map.of(coordinator.getLocalNode().getId(), coordinator.getClusterFormationState().getDescription())
```

Comment:
Would it be beneficial if the local node's view is expressed separately? Or put it the other way, the remote ones be expressed oner a new object in the representation? `remote` or something better named?

Reply:
Also, would we still want to have the cluster formation expressed as objects for future extension? (in case we'll want to add some structure)

ie: 
```    
"localNode": { "description" : " issues" },
"remote" : { 
       "node1" : { "description" : "issues" },
       "node2": { "description" : "other things"}
}
```

Reply:
I couldn't really think of any reason, so I put them together to simplify the response. Can you think of a reason it would be useful to have them separate?

Reply:
I was wondering if in large clusters it'll become a bit difficult to determine the local view amongst a list of remote views? Maybe I'm over-optimising. 
Could you please update the PR description with the latest responses and let's move ahead with the simple solution for now (we could iterate afterwards to improve it - once we see it in live cases)